test/e2e-test/src/api.ts

```diff
@@ -17,6 +22,13 @@ export interface ValidationProps {
   testMessage: string;
 }
 
+export const convertUint8ArrayToString = (uInt8array: Uint8Array): string => {
```

Comment:
I think we can delete the function here because we have the same function in utils/ dir